gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/resources/organizations/environments/domains/BotDetectionsResource.java

```diff
@@ -52,6 +53,9 @@ public class BotDetectionsResource extends AbstractResource {
     @Autowired
     private BotDetectionServiceProxy botDetectionService;
 
+    @Autowired
+    private BotDetectionPluginService botDetectionPluginService;
```

Comment:
It is not used ? Did I miss somethings ?

Reply:
indeed, I forgot to use it in the create method